editions/tw5.com/tiddlers/filters/examples/matches Operator (Examples).tid

```diff
@@ -0,0 +1,8 @@
+title: matches Operator (Examples)
+tags: [[matches Operator]] [[Operator Examples]]
+
+Note: The first example can be done using the [[field operator|field Operator]], the second example shows one way this operater differs from field.
+The large conceptual difference between this operator and field is that for field you have to have the exact field contents, for this operator you just need to have list but the order of the items on the list isn't important.
```

Comment:
when doing list filters, the order of the items should **never** be relevant!

Reply:
yes, for this operator order isn't relevant. That is why this works for matching the lists in two fields where the field operator wouldn't work.